core/src/main/java/nl/nn/adapterframework/http/rest/ApiListener.java

```diff
@@ -286,6 +288,22 @@ public String getOperationId() {
 		return operationId;
 	}
 
+	@IbisDoc({"12", "Comma separated list of parameters passed as http header.", ""})
+	public void setHeaderParams(String headerParams) {
+		this.headerParams = headerParams;
+	}
+	public String getHeaderParams() {
+		return headerParams;
+	}
```

Comment:
If you are adding this I expect a validation check and only add the headers stated in this setter to be added to the messageContext